File: qb-engineer-server/qb-engineer.core/Models/CustomerReturnListItemModel.cs
namespace QBEngineer.Core.Models;

public record CustomerReturnListItemModel(
    int Id,
    string ReturnNumber,
    int CustomerId,
    string CustomerName,
    int OriginalJobId,
    string OriginalJobNumber,
    int? ReworkJobId,
    string? ReworkJobNumber,
    string Status,
    string Reason,
    DateTimeOffset ReturnDate,
    DateTimeOffset CreatedAt);

File: qb-engineer-ui/src/app/features/customer-returns/models/customer-return-list-item.model.ts
export interface CustomerReturnListItem {
  id: number;
  returnNumber: string;
  customerId: number;
  customerName: string;
  originalJobId: number;
  originalJobNumber: string;
  reworkJobId: number | null;
  reworkJobNumber: string | null;
  status: string;
  reason: string;
  returnDate: Date;
  createdAt: Date;
}
